### ISSUE TITLE
Would like files that I change that are in $JENKINS_HOME to persist across container restarts

### ISSUE THREAD

**Issue by wicksy** [ministryofjustice/opg-docker#23]
At the moment /usr/local/bin/jenkins.sh copies in from the ref area regardless. Would be nice to have another environment variable that controls whether this copy happens or not. For example, once the initial instantiation of your container is done and reference data is copied, set an override variable so that subsequent "compose up" events skip the copy. Since the data is on a VOLUME changes (to jobs, users, etc) should persist. 

Tested out with this addition to environment.sh:

```
# Override copy of jenkins reference files over jenkins home
JENKINS_COPY_FROM_REF=no
```

and this minor mod to /usr/local/bin/jenkins.sh:

```
export -f copy_reference_file
if [[ $JENKINS_COPY_FROM_REF != "no" && $JENKINS_COPY_FROM_REF != "NO" ]] ; then
  echo "--- Copying files at $(date)" >> $COPY_REFERENCE_FILE_LOG
  find /usr/share/jenkins/ref/ -type f -exec bash -c "copy_reference_file '{}'" \;
fi
```

Leave the variable unset (or set to `YES`) by default and allow people to override when they want.
